.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -261,7 +261,7 @@ enum class TestType(val unitTests: Boolean = true, val functionalTests: Boolean
 enum class PerformanceTestType(val taskId: String, val timeout: Int, val defaultBaselines: String = "", val extraParameters: String = "", val hasRerunner: Boolean = true) {
     test("PerformanceTest", 420, "defaults"),
     experiment("PerformanceExperiment", 420, "defaults"),
-    flakinessDetection("FlakinessDetection", 420, "flakiness-detection-commit", hasRerunner = false),
+    flakinessDetection("FlakinessDetection", 420, "flakiness-detection-commit", "-PgithubToken=%github.ci.oauth.token%", hasRerunner = false),
```

Comment:
I think we can probably pass this parameter to all builds (`buildToolGradleParameters`) or all coordinator builds (`distributedPerformanceTestParameters`) so we only need to maintain this parameter in one place, WDYT?

Reply:
We can also pass the parameter to all performance coordinators.

Reply:
Then `distributedPerformanceTestParameters` would be a good place and this can be removed: https://github.com/gradle/gradle/blob/27529841ebd44f35d69b6c2abce65b8d2f5f07c6/.teamcity/Gradle_Check/configurations/PerformanceTestCoordinator.kt#L57

Reply:
Done.